modules/nlu/tsconfig.json

```diff
@@ -1,8 +1,7 @@
 {
   "compilerOptions": {
+    "lib": ["es2016"],
     "esModuleInterop": true,
-    "target": "es5",
-    "lib": ["es7"],
-    "typeRoots": ["./src/botpress.d.ts", "./src/global.d.ts", "./node_modules/@types", "./src/backend/typings.d.ts"]
```

Comment:
We should keep the same tsconfig (libraries) in each modules, otherwise it will be harder to update or move stuff in the future. If that change is for the best, could you apply it everywhere else ?